File: 6. Python/EX08.py
# Solicita a distância percorrida
dist = float(input("# Distancia percorrida: "))

# Solicita o combustível gasto
comGas = float(input("# Combustivel gasto: "))

# Calcula o consumo médio
conMed = dist / comGas

# Exibe o consumo médio formatado com três casas decimais
print(f"# Consumo medio = {conMed:.3f}")

File: 4. Java/EX08.java
package Apostila;

import java.util.Locale;
import java.util.Scanner;

public class EX08
{
    public static void main(String[] args)
    {
        // Declara variáveis
        int dist;
        double comGas, conMed;
        
        // Define localização
        Locale.setDefault(Locale.US);
        
        // Cria Scanner
        try (Scanner ent = new Scanner(System.in))
        {
            // Entrada de dados
            System.out.print("# Distancia percorrida: ");
            dist = ent.nextInt();
            
            System.out.print("# Combustivel gasto: ");
            comGas = ent.nextDouble();
        }
        
        // Calcula o consumo médio
        conMed = dist / comGas;
        
        // Exibe o consumo médio formatado
        System.out.println("# Consumo medio = " + String.format("%.3f", conMed));        
    }
}
